dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/utils/DagHelper.java

```diff
@@ -565,20 +554,40 @@ public static boolean haveConditionsAfterNode(String parentNodeCode, List<TaskNo
         return false;
     }
 
+
     /**
      * is there have blocking node after the parent node
      */
     public static boolean haveBlockingAfterNode(String parentNodeCode,
                                                 DAG<String,TaskNode,TaskNodeRelation> dag) {
+        return haveSubAfterNode(parentNodeCode, dag, true, TaskConstants.TASK_TYPE_BLOCKING);
+    }
+
+    /**
+     * is there have all node after the parent node
+     */
+    public static boolean haveAllNodeAfterNode(String parentNodeCode,
```

Comment:
Do you want to mean have any node?

Reply:
A node with child nodes in the process. The conditions for submitting child nodes when the current node fails are: there are child nodes and the failure strategy is continue. @ruanwenjun

Reply:
@WangJPLeo I mean that it better to rename this method name to `haveAnyNodeAfterNode`, since I find you just want to check if there exist any node after the parentNodeCode.

Reply:
ok thanks, i see what you mean.